### DISCUSSION TITLE
command 'github.copilot.generate' not found

### DISCUSSION THREAD

**Discussion by Zephxr** [community/community#116490] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

When I press shift + enter, I get this error while running the extension in vs code.  
![image](https://github.com/community/community/assets/71910018/c225f917-1bf8-49a5-b8ba-6b0c713fabf7)

I am not on a VPN or running it from behind a firewall.
There is no notification asking me to verify again.
The extension was running fine previously.
It will still provide autocomplete suggestions, just not allow me to generate batches of code with shift + enter and a few other things.
I have tried uninstalling the extension and VSCode, neither worked.
The extensions works as intended on my laptop under the same account.

**Comment by vital487** [comment]
I had this bug, But then I looked and the extension was asking me to reload VS Code to apply an update. After reloading VS Code it started working.

**Reply by Zephxr** [reply]
I have looked everywhere.  There is no notification or anything asking me to restart VS Code.  I have uninstalled both VS Code and the extension and reinstalled as well.  My guess is it has hidden files somewhere that don't go away on uninstall that got corrupted and are causing issues.

**Comment by Zephxr** [comment, marked as answer]
Fixed by removing all files related to the extension, by default found at Users\%USER%\.vscode\extensions

Source: https://stackoverflow.com/questions/36746857/completely-uninstall-vs-code-extensions
